docs/en/connector-v2/source/InfluxDB.md

```diff
@@ -0,0 +1,114 @@
+# InfluxDB
```

Comment:
Hi, In https://github.com/apache/incubator-seatunnel/pull/2625 we  have redefined the format of the document. Please modify the document according to the current latest document format.

Reply:
ok, I have modified it in new commit.